<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 205, 187);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Goudy Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>Secured Drone Communication</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>761</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Ctrl_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(237, 188, 160);
font: italic 10pt &quot;Goudy Old Style&quot;;
	
color: rgb(0,0,0);
border: 1px solid;
border-radius: 15px;
}
QPushButton:hover {
background-color: rgb(255, 233, 206);
color:rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>Establish d2d Connections</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ctrl_btn_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(237, 188, 160);
font: italic 10pt &quot;Goudy Old Style&quot;;
	
color: rgb(0,0,0);
border: 1px solid;
border-radius: 15px;
}
QPushButton:hover {
background-color: rgb(255, 233, 206);
color:rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>Establish d2g Connections</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
